widget/embedded/src/types/config.ts

```diff
@@ -237,6 +238,7 @@ export type WidgetConfig = {
   title?: string;
   walletConnectProjectId?: string;
   trezorManifest?: TrezorManifest;
+  tonconnectManifest?: string;
```

Comment:
The only configuration related to the TON Connect manifest is the manifestUrl. While the TON Connect SDK accepts multiple options for initialization, the only one directly tied to the manifest is `manifestUrl`. I believe `manifestUrl` is the key property we need to expose externally for all TON Connect-compatible providers.

As for `tonconnect/ui`, it can accept additional options that might be useful for users, which we can expose as a separate object. However, even in this case, the only property related to the manifest remains `manifestUrl`.

Reply:
Thanks. so i suggest this structure, it will be more flexible for future changes: 
```ts
{ 
tonConnect: {
    manifestUrl: string
  }
}
```